source/MaterialXFormat/Environ.cpp

```diff
@@ -0,0 +1,59 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXFormat/Environ.h>
+
+#include <MaterialXCore/Library.h>
+#include <MaterialXCore/Util.h>
+
+#if defined(_WIN32)
+#include <windows.h>
+#endif
+
+namespace MaterialX
+{
+
+string getEnviron(const string& name)
+{
+#if defined(_WIN32)
+    DWORD size = GetEnvironmentVariable(name.c_str(), nullptr, 0);
+
+    if (size != 0 && size != ERROR_ENVVAR_NOT_FOUND) 
+    {
+        std::unique_ptr<char[]> buffer(new char[size]);
+        GetEnvironmentVariable(name.c_str(), buffer.get(), size);
+        return string(buffer.get());
+    }
+
+    return EMPTY_STRING;
```

Comment:
This return of EMPTY_STRING can now be moved outside of the two platform-specific paths, in order to make the code slightly more streamlined.

Reply:
Will do